web_monitoring/differs.py

```diff
@@ -51,11 +52,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    obj = web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
```

Comment:
There were a couple of reasons:
1. I think it's better if we only return the result part of what we get from the API which was already implemented using `query_result` (which, I think, was also the motivation behind the function itself).
2. In case we add more diff functions which utilise the results from the PF diff, it would make sense to make use of the class methods in `pagefreezer.py`.